examples/fluent-bit/grafana-loki/README.md

```diff
@@ -0,0 +1,24 @@
+### FireLens Example: Logging to Grafana Loki with Fluent Bit
+
+For documentation on sending your FireLens monitored log data to Grafana Loki, see: [Fluent Bit and FireLens documentation](https://docs.fluentbit.io/manual/v/1.9-pre/pipeline/outputs/loki). Be aware there is a separate Golang output plugin provided by [Grafana](https://grafana.com/docs/loki/latest/clients/fluentbit/) with different configuration options. The Loki data source provides access to Loki, Grafana’s log aggregation system.
```

Comment:
> [Fluent Bit and FireLens documentation]

Since this is now the official upstream Fluent Bit docs, let's say that. That link does not contain info on FireLens, which is an AWS product. 

may be: "Fluent Bit Plugin Documentation"